browser.js

```diff
@@ -1,27 +1,7 @@
-var createHmac = require('create-hmac')
-var MAX_ALLOC = Math.pow(2, 30) - 1 // default in iojs
-
-exports.pbkdf2 = pbkdf2
-function pbkdf2 (password, salt, iterations, keylen, digest, callback) {
-  if (typeof digest === 'function') {
-    callback = digest
-    digest = undefined
-  }
-
-  checkParameters(iterations, keylen)
-
-  if (typeof callback !== 'function') {
-    throw new Error('No callback provided to pbkdf2')
-  }
-
-  setTimeout(function () {
-    var result = pbkdf2Sync(password, salt, iterations, keylen, digest)
-
-    callback(null, result)
-  })
-}
+exports._createHmac = require('create-hmac/browser')
```

Comment:
so the idea behind the /browser is so we can test the browser version in node. 

Since as far as I can tell, the actual outcome of this code is the same result as the must less complex situation of having a `var createHmac = require('create-hmac')` here I don't feel like we are gaining anything.